### PR TITLE
Idvalidation

### PR DESCRIPTION
**Pull request by aimanzi**
ID validation homework

### CODE REVIEW

File: Email validation/java/script.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        domaincounter = i; // the length of the domain after the @ ;
+    }
+    if (domaincounter > 63)
+        return false;
+    for (let i = strcounter1 + 1; i <= domaincounter; i++) { //hyphen -, provided that it is not the first or last character;
+        if (arr[0] == "-" || arr[domaincounter] == "-")
+            return false;
+    }
+    for (let i = strcounter1 + 1; i <= domaincounter; i++) { //digits 0 to 9, provided that top-level domain names are not all-numeric;
+        for (let x = 0; x <numbers.length; x++) {
```
Comment (EliasSalom) [inline Email validation/java/script.js:45]: there is a `number` variable and I think you forgot to define it, please review the code again, I received a `numbers is not defined` error

File: I.D Validation/index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/java/style/style.css">
+</head>
+
+<body>
+    <div>Enter your full name :<input type="text" placeholder="Name"  id="name"></div><br>
```
Comment (EliasSalom) [inline I.D Validation/index.html:13]: I think I have been saying many times that we are never let the text free, the text must be inside some element, please review the code again and fix this issue

**Review by EliasSalom — CHANGES_REQUESTED** [review]
please review my comments and fix all the issues and push a new code

File: I.D Validation/index.html (hunk 2 above)
Comment (aimanzi) [reply I.D Validation/index.html]: i have fixed thr uncorrect issues by adding <label> element before the Text

File: Email validation/java/script.js (hunk 1 above)
Comment (aimanzi) [reply Email validation/java/script.js:45]: i have fix the issues ,
the variable "numbers" wase undefined ;

**Review by EliasSalom — APPROVED** [review]
good work

**Pull request merged**